synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -217,7 +229,12 @@ StateBone_Context::load_settings()
 		if(settings.get_value("bone.bone_width",value) && value!="")
 			set_bone_width(Distance(atof(value.c_str()),App::distance_system));
 		else
-			set_bone_width(Distance(1,App::distance_system)); // default width
+			set_bone_width(Distance(6.6,App::distance_system)); // default width
```

Comment:
as 6.6 is used more than 2 times, I suggest to make it a constant value that you refer to it when it concerns.

Reply:
Btw, is it right to use `App::distance_system`? If user changed the unit system, wouldn't it change the default width?
Is 6.6 what? `Distance::SYSTEM_UNITS` ? `Distance::SYSTEM_PIXELS`? `Distance::SYSTEM_INCHES`?

`App::distance_system` is the system unit user chose.

Reply:
It's system_pixels. Will make the change.

Reply:
This issue is still present in current code.

Reply:
I decided to have it's `distance_system` fixed to `SYSTEM_UNITS`.